protocol/x/clob/keeper/process_operations.go

```diff
@@ -394,6 +367,35 @@ func (k Keeper) PersistOrderRemovalToState(
 				orderToRemove.GetBaseQuantums(),
 			)
 		}
+	// TODO - uncomment when reduce only orders are enabled. Order Removals of this type will fail ValidateBasic.
```

Comment:
Suggestion: add the ticket?

```suggestion
	// TODO (CLOB-877) - uncomment when reduce only orders are enabled. Order Removals of this type will fail ValidateBasic.
```